x-pack/plugins/security_solution/public/common/components/autocomplete/field_value_match_any.tsx

```diff
@@ -64,14 +67,8 @@ export const AutocompleteFieldMatchAnyComponent: React.FC<AutocompleteFieldMatch
     onChange(newValues);
   };
 
-  const onSearchChange = (searchVal: string) => {
-    if (updateSuggestions != null) {
-      updateSuggestions({
-        fieldSelected: selectedField,
-        value: searchVal,
-        patterns: indexPattern,
-      });
-    }
+  const handleSearchChange = (searchVal: string) => {
```

Comment:
As above, can leverage a `useCallback` for these handlers to maintain references as passed into the below `EuiComboBox` component.